### PR TITLE
Upgrade to Leap 42.1, and fix apache2 starting problem

### PR DESCRIPTION
**Pull request by wnereiz**
Tested, it can be built and worked without problem.

### CODE REVIEW

File: docker/webui/run_openqa.sh [hunk 1]
```diff
@@ -9,5 +9,5 @@ dbus-daemon --system --fork
 start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler &
 start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets &
 start_daemon -u geekotest /usr/share/openqa/script/openqa gru -m production run &
-rcapache2 start
+apache2ctl start
```
Comment (coolo) [inline docker/webui/run_openqa.sh]: I think this should use systemctl

**Comment by okurz** [comment]
nice. BTW, is there some kind of CI system for docker files?

File: docker/webui/run_openqa.sh (hunk 1 above)
Comment (aaannz) [inline docker/webui/run_openqa.sh]: Will it work? This is started from within docker container and AFAIK "it's complicated" between them.

Comment (coolo) [inline docker/webui/run_openqa.sh]: I don't know - I was just thinking aloud. But now that you mention it (not sure why @wnereiz ignored my thought :), the rest of this bash script doesn't look like it's handled by systemd either.

So just merge and take the blame ,)

**Pull request merged**